nuget/lock-file/packages.lock.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
This file is intentionally empty.  It just needs to be present to trigger the update.